virtual_rainforest/core/grid.py

```diff
@@ -113,7 +106,7 @@ def make_square_grid(
                 prototype, xoff=scale_factor * x_idx, yoff=scale_factor * y_idx
             )
 
-    return GridStructure(ids=list(range(len(cell_list))), polygons=cell_list)
+    return (tuple(range(len(cell_list))), tuple(cell_list))
```

Comment:
You don't need the external parenthesis. A `return` statement with elements separated by commas automatically results in a tuple.
```suggestion
    return tuple(range(len(cell_list))), tuple(cell_list)
```